<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDubb</class>
 <widget class="QMainWindow" name="QDubb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDubb</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonLoad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load a File</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>641</width>
      <height>84</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPrevious">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNext">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCounter">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>QDubb</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
